explorer/src/logging.ts

```diff
@@ -10,11 +10,16 @@ const options: Parameters<typeof pino>[0] = {
     paths: ['req.headers', 'res.headers'],
   },
 }
-if (process.env.EXPLORER_DEV) {
-  options.prettyPrint = { colorize: true }
-  options.level = 'debug'
-} else if (process.env.NODE_ENV === 'test') {
-  options.level = 'silent'
+
+switch (process.env.NODE_ENV) {
```

Comment:
Can we switch on `ExplorerConfig` instead? We shouldn't use environment variables anywhere other than in `config.ts`

Reply:
Maybe, looks like it'd require a bit of refactoring.

Reply:
https://www.pivotaltracker.com/story/show/172345097